locale/fi/LC_MESSAGES/django.po

```diff
@@ -3443,6 +3451,58 @@ msgstr ""
 "Ilmoitusviestien vastaanottajat. Esimerkki: matti@example.com,maija@example."
 "com"
 
+msgctxt "Search class"
+msgid "Plot search"
+msgstr "Tonttihaku tai kilpailu"
+
+msgctxt "Search class"
+msgid "Other"
+msgstr "Muu kilpailu tai haku"
+
+msgctxt "Information state"
+msgid "Checked"
+msgstr "Tarkistettu"
+
+msgctxt "Information state"
+msgid "Not checked"
+msgstr "Ei ole tarkistettu"
+
+msgctxt "Information state"
+msgid "Not needed"
+msgstr "Ei tarvetta"
+
+msgctxt "Information state"
+msgid "Requires further action"
+msgstr "Tarvitsee lisätietoja"
```

Comment:
Ei suora käännös mutta varmaan kertoo juuri sen mitä pitää?

Reply:
Näin arvelin